gtk/src/toga_gtk/widgets/button.py

```diff
@@ -9,12 +9,11 @@ def create(self):
         self.native = Gtk.Button()
         self.native.interface = self.interface
 
-        self.native.connect("show", lambda event: self.rehint())
+        self.native.connect("show", lambda event: self.refresh())
```

Comment:
I suspect these event handlers may no longer be necessary. @freakboy3742: Do you know what they're for?

Reply:
The GTK "Show" event is the event that fires when the widget is first visible on screen; as I recall, it was possible (at least, previously, anyway), for the layout algorithm to fire when the widget hadn't yet been made visible, which resulted in initial sizes of (0,0). This call forced a rehint when the widget was made visible, to ensure the first visible size was sensible.

However, this *may* now be reundant, as displaying the container will cause a layout, and (I think?) implicitly rehint of all widgets.